source/patterns/01-molecules/inline-list/inline-list.twig

```diff
@@ -6,6 +6,10 @@
   {%- set attributes = attributes|merge({class: attributes.class|merge(['inline-list--small']) }) -%}
 {%- endif -%}
 
+{%- if alternativeSeparators|default(false) -%}
+  {%- set attributes = attributes|merge({class: attributes.class|merge(['inline-list--semi-colon-separated']) }) -%}
```

Comment:
Class needs renaming.

Reply:
I wondered about this. Currently have kept this class naming what it does, so if another separator is required, a class can be created for that, and this line changed to apply that different class. Seems more flexible than just having one alternate, the implementation of which would be changed.

Reply:
Might not always be semi-colons depending on the locale...

Reply:
Okay, let's keep it simple, will change it.